Classes/UT3Hellbender.uc

```diff
@@ -428,20 +428,27 @@ defaultproperties
     VehiclePositionString="in a Hellbender"
     VehicleNameString="UT3 Hellbender"
     HornSounds(0)=Sound'UT3A_Vehicle_Hellbender.Sounds.A_Vehicle_Hellbender_Horn01'
-    GroundSpeed=700.000000
+    GroundSpeed=800.000000 //700
     SoundVolume=255
-
-    DrawScale=1.0
+    TransRatio=0.18 //0.11
+    
+	DrawScale=0.9
+    
     CollisionRadius=219
-    HeadlightCoronaOffset(0)=(X=77.5,Y=27.5,Z=52.5)
-    HeadlightCoronaOffset(1)=(X=77.5,Y=-27.5,Z=52.5)
-    HeadlightCoronaOffset(2)=(X=77.5,Y=25,Z=41)
-    HeadlightCoronaOffset(3)=(X=77.5,Y=-25,Z=41)
+    HeadlightCoronaOffset(0)=(X=68.5,Y=24.5,Z=47.5) //(X=77.5,Y=27.5,Z=52.5)
+    HeadlightCoronaOffset(1)=(X=68.5,Y=-24.5,Z=47.5)
+    HeadlightCoronaOffset(2)=(X=68.5,Y=22,Z=38) //(X=77.5,Y=-25,Z=41)
+    HeadlightCoronaOffset(3)=(X=68.5,Y=-22,Z=38)
     HeadlightCoronaMaterial=Material'EmitterTextures.Flares.EFlareOY'
-    HeadlightCoronaMaxSize=94
-    HeadlightProjectorMaterial=None
+    HeadlightCoronaMaxSize=82
+
+	HeadlightProjectorOffset=(X=78.0,Y=0,Z=50.5) //(X=82.5,Y=0,Z=55.5)
+	HeadlightProjectorRotation=(Yaw=0,Pitch=-1000,Roll=0)

```

Comment:
Also, this is clearly not exact: rotators in the Unreal Engine are in RUUs that have a range of 0-65535. So if it is close to -1000, then it should be set to -1024 (that's a round number).